app/src/main/res/values-night/colors.xml

```diff
@@ -2,6 +2,8 @@
 <resources>
     <color name="background">#1F1D2B</color>
 
+    <color name="primary_dark">#BE6C5ECF</color>
```

Comment:
why add new value to primary_dark   ????

Reply:
by wrong it should be primary_pressed